Cmug esacci oc chlor a (#2055)
ESMValGroup/ESMValTool: doc/sphinx/source/recipes/recipe_esacci_oc.rst

.. _recipes_<mynewrecipe>:
   
Ocean chlorophyll in ESMs compared to ESA-CCI observations.
===========================================================

Overview
--------

> This recipe compares monthly surface chlorophyll from CMIP models to ESA CCI ocean colour chlorophyll. The observations are the merged sensor geographic monthly L3S chlor_a data Sathyendranath et al (2019). Multiple models and different observational versions can be used by the script.

Reviewer: ```suggestion
This recipe compares monthly surface chlorophyll from CMIP models to ESA CCI ocean colour chlorophyll. The observations are the merged sensor geographic monthly L3S chlor_a data Sathyendranath et al. (2019). Multiple models and different observational versions can be used by the script.
```

> The recipe_esacci_oc.yml produces an image showing four maps. Each of these four maps show latitude vs longitude and the cube value is used as the colour scale. The four plots are: ESA-CCI-OC v5.0 chlorophyll, the CMIP6 model, the bias model-observation and the ratio model/ observations. The script also produces a scatter plot for all coordinates with the model on the x-axis and the observations on the y axis and a line of best fit with the parameter values given in the panel.

Reviewer: ```suggestion
The recipe_esacci_oc.yml produces an image showing four maps. Each of these four maps shows latitude vs longitude and the chlorophyll value. The four plots are: ESA-CCI-OC v5.0 chlorophyll, the CMIP6 model, the bias model-observation and the ratio model/observations. The script also produces a scatter plot for all coordinates with the model on the x-axis and the observations on the y axis and a line of best fit with the parameter values given in the panel.
```

Available recipes and diagnostics
---------------------------------

Recipes are stored in esmvaltool/recipes/

    * recipe_esacci_oc.yml

Diagnostics are stored in esmvaltool/diag_scripts/<mynewdiag>/

>     * <mynewdiag.py/.ncl/.r>: one line scription

Reviewer: ```suggestion
```

    * diagnostic_model_vs_obs.py

User settings in recipe
-----------------------

> #. Script <mynewdiag.py/.ncl/.r>

Reviewer: ```suggestion
#. Script diagnostic_model_vs_obs.py
```

   *Required settings for script*

   * xxx: zzz

   *Optional settings for script*

   *Required settings for variables*

   *Optional settings for variables*

   *Required settings for preprocessor*

   *Optional settings for preprocessor*

   *Color tables*

>    * list required color tables (if any) here

Reviewer: This should be populated with the corresponding variables or removed if not needed.

Variables
---------

* chlor_a (ocean, monthly mean, longitude, latitude, time)

Observations and reformat scripts
---------------------------------

> *Note: (1) obs4MIPs data can be used directly without any preprocessing;
> (2) see headers of reformat scripts for non-obs4MIPs data for download
> instructions.*
> 
> * http://dx.doi.org/10.5285/00b5fc99f9384782976a4453b0148f49
> 
>   *Reformat script:* <myreformatscript.py>

Reviewer: ```suggestion
ESACCI-OC (chlor_a - esmvaltool/cmorizers/obs/cmorize_obs_esacci_oc.py)
```

References
----------

> * Sathyendranath, S, et al (2019) An ocean-colour time series for use in climate studies: the experience of the Ocean-Colour Climate Change Initiative (OC-CCI). Sensors: 19, 4285. doi:10.3390/s19194285

Reviewer: ```suggestion
* Sathyendranath, S., et al. (2019), An ocean-colour time series for use in climate studies: the experience of the Ocean-Colour Climate Change Initiative (OC-CCI). Sensors: 19, 4285. doi:10.3390/s19194285.
```

> 

Reviewer: * ESACCI-OC dataset: http://dx.doi.org/10.5285/00b5fc99f9384782976a4453b0148f49

Example plots
-------------

.. _fig_mynewdiag_1:

> .. figure::  /recipes/figures/<mynewdiagnostic>/awesome1.png

Reviewer: An example figure would be great but is still missing.

   :align:   center

>    Add figure caption here.

Reviewer: ```suggestion
```
